third_party/pyth/p2w_autoattest.py

```diff
@@ -190,31 +190,32 @@ def find_and_log_seqnos(s):
 mapping_reload_interval_mins: 1 # Very fast for testing purposes
 min_rpc_interval_ms: 0 # RIP RPC
 max_batch_jobs: 1000 # Where we're going there's no oomkiller
-symbol_groups:
+default_attestation_conditions:
+  min_interval_secs: 30
+  price_changed_bps: 500
+name_groups:
```

Comment:
hacked the existing tests to include name groups

Reply:
I don't think leaving this empty is right. I think Tilt setup should be able to show that the name group settings are respected _by default_. Other than that, we should be able to confirm that this setting is respected in the log, ideally with `warn!` on name groups that were never used.